pkg/controllers/serviceexport_controller.go

```diff
@@ -122,7 +122,8 @@ func (r *ServiceExportReconciler) handleUpdate(ctx context.Context, log logr.Log
 
 	if createRequired || updateRequired {
 		// merge creates and updates (Cloud Map RegisterEndpoints can handle both)
-		upserts := append(changes.Create, changes.Update...)
+		upserts := changes.Create
+		upserts = append(changes.Create, changes.Update...)
```

Comment:
Should be either 
```
upserts := changes.Create
upserts = append(upserts, changes.Update...)
```
or:
```
upserts := append(changes.Create, changes.Update...)
```
If lint squawks about the second as you mentioned, let's go with the first.

Reply:
makes sense. Sorry, I missed it before.